src/test/clojure/cemerick/pomegranate_nrepl_test.clj

```diff
@@ -0,0 +1,30 @@
+(ns cemerick.pomegranate-nrepl-test
+  (:require [nrepl.core :as repl]
+            [nrepl.server :as server])
+  (:use clojure.test))
+
+(def port 7888)
+
+(defn nrepl-eval [code]
+  (with-open [conn (repl/connect :port port)]
+    (-> (repl/client conn 1000)
+        (repl/message {:op :eval :code code})
+        repl/response-values)))
+
+;; without this openjdk-7 fail in travis, but there's no exception
```

Comment:
if you're interested: https://travis-ci.org/razum2um/pomegranate/jobs/403479139
locally tried zulu and oracle jdk-7 and it worked without this catch, so I couldn't reproduce that fail.
feel free to comment if you understand the reason :)

Reply:
JDK 7 is ancient and even Clojure is dropping (finally) support for it in 1.10, so I wouldn't fret too much over JDK7-only issues.